### ISSUE TITLE
No Tar files extracted in preprocessing

### ISSUE THREAD

**Issue by alexwolson** [alexwolson/building-image-triplet-model#36]
error log:

Lmod is automatically replacing "gcc/12.3" with "intel/2023.2.1".

Due to MODULEPATH changes, the following have been reloaded:
  1) flexiblas/3.3.1     2) openmpi/4.1.5

warning: The `tool.uv.dev-dependencies` field (used in `pyproject.toml`) is deprecated and will be removed in a future release; use `dependency-groups.dev` instead
warning: The `requires-python` specifier (`~=3.12`) in `building-image-triplet-model` uses the tilde specifier (`~=`) without a patch version. This will be interpreted as `>=3.12, <4`. Did you mean `~=3.12.0` to constrain the version as `>=3.12.0, <3.13`? We recommend only using the tilde specifier with a patch version to avoid ambiguity.
Using CPython 3.12.4 interpreter at: /cvmfs/soft.computecanada.ca/easybuild/software/2023/x86-64-v4/Compiler/gcccore/python/3.12.4/bin/python3
Creating virtual environment at: .venv
Activate with: source .venv/bin/activate
warning: The `tool.uv.dev-dependencies` field (used in `pyproject.toml`) is deprecated and will be removed in a future release; use `dependency-groups.dev` instead
warning: The `requires-python` specifier (`~=3.12`) in `building-image-triplet-model` uses the tilde specifier (`~=`) without a patch version. This will be interpreted as `>=3.12, <4`. Did you mean `~=3.12.0` to constrain the version as `>=3.12.0, <3.13`? We recommend only using the tilde specifier with a patch version to avoid ambiguity.
Resolved 99 packages in 38ms
   Building building-image-triplet-model @ file:///local/scratch/awolson.3627486.0/building-image-triplet-model
Downloading setuptools (1.1MiB)
Downloading scipy (35.6MiB)
Downloading nvidia-cufft-cu12 (201.7MiB)
Downloading nvidia-cusparselt-cu12 (143.1MiB)
Downloading triton (241.4MiB)
Downloading nvidia-cusparse-cu12 (197.8MiB)
Downloading nvidia-nccl-cu12 (179.9MiB)
Downloading nvidia-cusolver-cu12 (122.0MiB)
Downloading nvidia-nvjitlink-cu12 (20.1MiB)
Downloading nvidia-cuda-cupti-cu12 (13.2MiB)
Downloading h5py (5.2MiB)
Downloading numpy (15.8MiB)
Downloading nvidia-cuda-nvrtc-cu12 (23.5MiB)
Downloading torchvision (6.9MiB)
Downloading nvidia-cublas-cu12 (346.6MiB)
Downloading scikit-learn (12.3MiB)
Downloading pandas (12.1MiB)
Downloading nvidia-curand-cu12 (53.7MiB)
Downloading nvidia-cudnn-cu12 (634.0MiB)
Downloading networkx (1.9MiB)
Downloading sympy (5.9MiB)
Downloading pydantic-core (2.0MiB)
Downloading black (1.7MiB)
Downloading hf-xet (3.2MiB)
Downloading aiohttp (1.7MiB)
Downloading pygments (1.2MiB)
Downloading torch (731.1MiB)
Downloading timm (2.4MiB)
Downloading pillow (4.3MiB)
Downloading sqlalchemy (3.2MiB)
Downloading wandb (20.4MiB)
      Built building-image-triplet-model @ file:///local/scratch/awolson.3627486.0/building-image-triplet-model
 Downloading pygments
 Downloading aiohttp
 Downloading pydantic-core
 Downloading black
 Downloading setuptools
 Downloading timm
 Downloading networkx
 Downloading sqlalchemy
 Downloading hf-xet
 Downloading pillow
 Downloading h5py
 Downloading torchvision
 Downloading nvidia-cuda-cupti-cu12
 Downloading numpy
 Downloading nvidia-nvjitlink-cu12
 Downloading pandas
 Downloading sympy
 Downloading nvidia-cuda-nvrtc-cu12
 Downloading scikit-learn
 Downloading nvidia-curand-cu12
 Downloading nvidia-cusolver-cu12
 Downloading wandb
 Downloading nvidia-cusparselt-cu12
 Downloading nvidia-nccl-cu12
 Downloading nvidia-cusparse-cu12
 Downloading nvidia-cufft-cu12
 Downloading scipy
 Downloading nvidia-cublas-cu12
 Downloading triton
 Downloading nvidia-cudnn-cu12
 Downloading torch
Prepared 98 packages in 2m 14s
warning: Failed to hardlink files; falling back to full copy. This may lead to degraded performance.
         If the cache and target directories are on different filesystems, hardlinking may not be supported.
         If this is intentional, set `export UV_LINK_MODE=copy` or use `--link-mode=copy` to suppress this warning.
Installed 98 packages in 28.09s
 + aiohappyeyeballs==2.6.1
 + aiohttp==3.13.1
 + aiosignal==1.4.0
 + alembic==1.17.0
 + annotated-types==0.7.0
 + attrs==25.4.0
 + black==25.1.0
 + building-image-triplet-model==0.0.1 (from file:///local/scratch/awolson.3627486.0/building-image-triplet-model)
 + certifi==2025.10.5
 + charset-normalizer==3.4.4
 + click==8.3.0
 + colorlog==6.10.1
 + docker-pycreds==0.4.0
 + filelock==3.20.0
 + flake8==7.2.0
 + frozenlist==1.8.0
 + fsspec==2025.9.0
 + gitdb==4.0.12
 + gitpython==3.1.45
 + greenlet==3.2.4
 + h5py==3.12.1
 + hf-xet==1.2.0
 + huggingface-hub==0.36.0
 + idna==3.11
 + iniconfig==2.3.0
 + isort==6.0.1
 + jinja2==3.1.6
 + joblib==1.5.2
 + lightning==2.5.5
 + lightning-utilities==0.15.2
 + mako==1.3.10
 + markdown-it-py==4.0.0
 + markupsafe==3.0.3
 + mccabe==0.7.0
 + mdurl==0.1.2
 + mpmath==1.3.0
 + multidict==6.7.0
 + mypy-extensions==1.1.0
 + networkx==3.5
 + numpy==2.2.6
 + nvidia-cublas-cu12==12.4.5.8
 + nvidia-cuda-cupti-cu12==12.4.127
 + nvidia-cuda-nvrtc-cu12==12.4.127
 + nvidia-cuda-runtime-cu12==12.4.127
 + nvidia-cudnn-cu12==9.1.0.70
 + nvidia-cufft-cu12==11.2.1.3
 + nvidia-curand-cu12==10.3.5.147
 + nvidia-cusolver-cu12==11.6.1.9
 + nvidia-cusparse-cu12==12.3.1.170
 + nvidia-cusparselt-cu12==0.6.2
 + nvidia-nccl-cu12==2.21.5
 + nvidia-nvjitlink-cu12==12.4.127
 + nvidia-nvtx-cu12==12.4.127
 + optuna==4.1.0
 + packaging==25.0
 + pandas==2.2.3
 + pathspec==0.12.1
 + pillow==11.1.0
 + platformdirs==4.5.0
 + pluggy==1.6.0
 + propcache==0.4.1
 + protobuf==6.33.0
 + psutil==7.1.1
 + pycodestyle==2.13.0
 + pydantic==2.12.3
 + pydantic-core==2.41.4
 + pyflakes==3.3.2
 + pygments==2.19.2
 + pytest==7.4.4
 + python-dateutil==2.9.0.post0
 + pytorch-lightning==2.5.5
 + pytz==2025.2
 + pyyaml==6.0.3
 + requests==2.32.5
 + rich==14.0.0
 + safetensors==0.6.2
 + scikit-learn==1.5.2
 + scipy==1.15.3
 + sentry-sdk==2.42.1
 + setproctitle==1.3.7
 + setuptools==80.9.0
 + six==1.17.0
 + smmap==5.0.2
 + sqlalchemy==2.0.44
 + sympy==1.13.1
 + threadpoolctl==3.6.0
 + timm==1.0.20
 + torch==2.6.0
 + torchmetrics==1.8.2
 + torchvision==0.21.0
 + tqdm==4.67.1
 + triton==3.2.0
 + typing-extensions==4.15.0
 + typing-inspection==0.4.2
 + tzdata==2025.2
 + urllib3==2.5.0
 + wandb==0.19.11
 + yarl==1.22.0
ERROR: No tar files were successfully extracted!

output log:

==========================================
Starting preprocessing job on g2.nibi.sharcnet at Wed 29 Oct 2025 10:41:33 AM EDT
Job ID: 3627486
==========================================
Configuration:
  TAR_SOURCE_DIR: /home/awolson/scratch/awolson/3d_street_view_archives/dataset_unaligned
  EXTRACT_DIR: /local/scratch/awolson.3627486.0/extracted_dataset
  OUTPUT_HDF5: /home/awolson/scratch/building-image-triplet-model/dataset.h5

[Wed 29 Oct 2025 10:41:34 AM EDT] Setting up environment...
[Wed 29 Oct 2025 10:41:34 AM EDT] Copying project to /local/scratch/awolson.3627486.0/building-image-triplet-model...
[Wed 29 Oct 2025 10:41:35 AM EDT] Creating virtual environment and installing dependencies...
[Wed 29 Oct 2025 10:44:19 AM EDT] Environment setup complete

[Wed 29 Oct 2025 10:44:19 AM EDT] Starting tar file extraction...

[Wed 29 Oct 2025 10:44:19 AM EDT] Extraction complete: 0 succeeded, 0 failed
